docs/sources/helm-charts/mimir-distributed/configure/configure-native-histograms-ingestion.md

```diff
@@ -23,18 +23,6 @@ mimir:
 
 > **Note:** Native histograms is an experimental feature of Grafana Mimir.
 
-## Configure Prometheus to write native histograms to Grafana Mimir
```

Comment:
Should we replace with reference to the new doc? @osg-grafana

Reply:
Please remove that subsection and add a cross-reference in a sentence that contains a `relref`.

```
To learn more about sending native histograms to Mimir or Grafana Cloud Metrics via Grafana Agent or Prometheus, see [Scrape and send native histograms with Grafana Agent]({{< relref "#TODO"}}) or [Scrape and send native histograms with Prometheus]({{< relref "TODO"}}).
```